docs/Course Wiki/PHYS Course Offerings/PHYS214.md

```diff
@@ -39,14 +39,14 @@ In the past, this course has been taught by Professors Wagner and Covey.
 
 ## Course Tips
 
-The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study.
+The key to this course is the equation sheet. Learn how to use all the equations and the homework/exams should be fairly simple. There are also provided practice exams released prior to the real thing. They are usually very similar to the actual exam, so they are the best tool to study. The difficulty ranking of problems you will encounter is: Discussion Worksheets, smartPhysics Homework, Exams. So if you understand the discussions and homeworks, you will do fine on exams.
```

Comment:
Not sure what the order of difficulty is, but change to something like "Out of the discussion worksheets, smartPhysics homeworks, and exams, discussion worksheets are the most difficult and exams are the least difficult."  If that's what you mean.